Patches/OpenCV3/Patch.cmake

```diff
@@ -14,3 +14,11 @@ file(COPY ${OpenCV_patch}/graphcuts.cpp
 file(COPY ${OpenCV_patch}/OpenCVFindLibsVideo.cmake
   DESTINATION ${OpenCV_source}/cmake
 )
+
```

Comment:
perfect, thanks. I forgot to check the individual commits. I am hoping a dashboard will run, but look unlikely. I will merge in a bit.